src/cromshell/utilities/command_setup_utils.py

```diff
@@ -17,5 +17,17 @@ def resolve_workflow_id_and_server(workflow_id: str, cromshell_config) -> str:
     http_utils.set_and_check_cromwell_server(
         config=cromshell_config, workflow_id=resolved_workflow_id
     )
+    set_workflow_id(workflow_id=resolved_workflow_id, cromshell_config=cromshell_config)
 
     return resolved_workflow_id
+
+
+def set_workflow_id(workflow_id: str, cromshell_config) -> None:
```

Comment:
I'm concerned about this.
What about commands that support multiple workflow IDs?

Reply:
resolved during code review meeting.